locale/pl/LC_MESSAGES/client.po

```diff
@@ -95,7 +95,7 @@ msgstr "Przykro nam o tym słyszeć. Jeśli nie ufasz już temu urządzeniu, to
 
 #: .es5/scripts/views/settings/clients.js:52
 msgid "Last active %(translatedTimeAgo)s"
-msgstr ""
+msgstr "Ostatnia aktywność: %(translatedTimeAgo)s"
```

Comment:
Even if the translation is present at more or less accurate right now this doesn't change a thing, moment/moment project is very different from CLDR to the point that I consider this example as very bad joke. If not sure you can ask an, as, ast, bn_BD, bn_IN, dsb, es_*, ff, fy_NL, ga, ga_IE, gu, gu_IN and so many other locales that I didn't want to waste time checking…

> Yea this is a stop gap solution, we have several problems with current string extraction, such as plural forms, extra info about the string, etc. Hopefully we can gather those issues together and develop a better way to manage all these strings in the next couple of months.

This doesn't explain why and how it was chosen and why it was integrated in a such way. The relative dates issue was reported long time before and mostly ignored. I don't believe in next couple months solution at all.